docs/howtos/intercepts.md

```diff
@@ -65,7 +65,7 @@ network telepresence, and remote mounts must be made relative to a specific moun
 
 4. Intercept all traffic going to the application in your cluster:
     ```
-    telepresence intercept <workload-name> --port [<local-port>][:<remote-port>] --env-file <path-to-env-file>`.
+    telepresence intercept &lt;workload-name&gt; --port [&lt;local-port&gt;][:&lt;remote-port&gt;] --env-file &lt;path-to-env-file&gt;`.
```

Comment:
Yeah, I made that change after a build failure from docusaurus suggesting that this was treated as mdx, which it shouldn't be given the extension. I'll investigate this further, because the preview on GH should be as good as it can be.